docs/Installation/Installation.md

```diff
@@ -90,48 +89,39 @@ Repeat this procedure for all three controllers.
 @TODO
 
 
-<a id="man-container"></a>
-## Using the management image for the installer script
+<a id="kubernetes-cluster"></a>
+### Setup the Kubernetes cluster and deploy the OpenvCloud system containers
 
-To have all the necessary requirements to use the installer to setup the cluster, you can use the management docker image.
+This will create a Kubernetes cluster, and deploy all OpenvCloud system containers needed to manage an OpenvCloud cluster.
 
-The Docker image is available on Docker Hub: https://hub.docker.com/r/openvcloud/management/.
+One of these containers is the management container, through which you will be able to check the status of all other containers;discussed next.
 
-Install Docker on the first controller node:
```

Comment:
We need to keep this part of the instalation as the controllers come withouth docker
However you can update it with the latest version of docker like we use in baldur
https://git.gig.tech/openvcloud/baldur/blob/master/baldur/kubernetes.py#L53